hide-items.css

```
@@ -55,7 +55,13 @@ div.bar-icon { /* 隐藏聊天页面上侧的语音通话，视频通话，屏
     display: none; 
 }
 
-html > body.q-theme-tokens-light > div.lite-tools-vue-component.vue-component > div.container > div.nav-bar-container.lite-tools-vue-component.vue-component > div > div.nav-item > div > div.nav-item-container.lite-tools-vue-component.vue-component:last-child {
+div#app > div.container:nth-of-type(2) > div.nav-bar-container.vibrancy.lite-tools-vue-component.vue-component:nth-of-type(1) > div:nth-of-type(2) > div.wifiphoto-btn {
+/*  修复隐藏旧版(29804)文件管理器左下方的“导入手机相册”工具栏失效  */
+    display: none;
+}
+
+
+html > body.q-theme-tokens-light > div.lite-tools-vue-component.vue-component > div.container > div.nav-bar-container.lite-tools-vue-component.vue-component > div > div.nav-item > div > div.nav-item-container.lite-tools-vue-component.vue-component:nth-of-type(2) {
```

`.q-theme-tokens-light` 仅在浅色模式存在，其余同上

```suggestion
html > body > div > div.container > div.nav-bar-container > div > div.nav-item > div > div.nav-item-container:nth-of-type(2) {
```